### DISCUSSION TITLE
Usb port on wand for programming

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#23] [Ideas]
Do either of you think there is value in adding a usb port to the wand for programming new code for future purposes? I finished my wand, you guys keep killing it on the new code so I have to open it up again lol

**Comment by gpstar81** [comment]
> Do either of you think there is value in adding a usb port to the wand for programming new code for future purposes? I finished my wand, you guys keep killing it on the new code so I have to open it up again lol

I thought about it, but I figured it's only 8 screws to pop open the wand, it takes less than a minute to open it up.

The pack on the other hand, I keep a USB cable on the Mega and run it up to the powercell behind the door. Then I use a usb extension to reach my computer to update the pack code.

**Comment by nomakewan** [comment]
Old discussion but in case you haven't seen it, one of our members created a really awesome solution for this: https://www.etsy.com/listing/1756220009

We'll be adding a link to it to our documentation as our official recommendation shortly.

**Comment by DustinGrau** [comment]
Link just added for v5.3.4
